splunk_add_on_ucc_framework/commands/init.py

```diff
@@ -103,9 +104,38 @@ def _generate_addon(
         _f.write(global_config_rendered_content)
     package_path = os.path.join(generated_addon_path, "package")
     os.makedirs(package_path)
-    package_license_path = os.path.join(package_path, "LICENSE.txt")
-    with open(package_license_path, "w") as _f:
-        pass
+    package_license_dir = os.path.join(package_path, "LICENSES")
+    os.makedirs(package_license_dir)
+
+    if add_license:
+        if add_license in [
+            "Apache License 2.0",
+            "MIT License",
+            "SPLUNK PRE-RELEASE SOFTWARE LICENSE AGREEMENT",
+        ]:
```

Comment:
We could also outsource this logic to `argparse.choices` (https://docs.python.org/3/library/argparse.html#choices), what do you think?

Reply:
Updated the code.